regression.py

```diff
@@ -89,6 +89,45 @@ def testADF_ADF2013_01_dvb_gopt_b_unconverged_adfout(logfile):
     """An unconverged geometry optimization to test for empty optdone (see #103 for details)."""
     assert hasattr(logfile.data, 'optdone') and not logfile.data.optdone
 
+# def testADF_ADF2013_01_stopiter_dvb_sp_adfout(logfile):

```

Comment:
They don't run because something is very fishy with the ADF parser regarding the number of MOs or basis functions. Can I uncomment them but wrap them with skip decorators until the issue is resolved, or should I try and fix the parser first?

I agree that I'd rather not insert commented-out code at all.